src/models/room-state.js

```diff
@@ -182,7 +182,7 @@ RoomState.prototype.setStateEvents = function(stateEvents) {
             utils.forEach([member, sentinel], function(roomMember) {
                 roomMember.setMembershipEvent(event, self);
                 // this member may have a power level already, so set it.
-                const pwrLvlEvent = self.getStateEvents("m.room.power_levels", "");
+                const pwrLvlEvent = self.getStateEvents("m.room.powerLevels", "");
```

Comment:
apart from my request not to make unrelated changes in the same PR, this looks wrong.